Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY: test hh_autoload composer
+
+COMPOSER=~/.cache/composer/composer.phar
+
+test: hh_autoload
+	./vendor/bin/hacktest tests
+
+lint: hh_autoload
+	./vendor/bin/hhast-lint
+
+autoload:
+	./vendor/bin/hh-autoload
+
+composer-fetch:
+	test -f $(COMPOSER) || mkdir -p `dirname $(COMPOSER)`
+	test -f $(COMPOSER) || wget -O $(COMPOSER) https://getcomposer.org/download/1.9.0/composer.phar
```

Comment:
1. use `curl` instead; wget is less frequently available (e.g. curl is a standard macos command)
2. don't hardcode the version here. We dont' want to have to change a makefile in every project every time